compiler/src/compiler.go

```diff
@@ -21,32 +20,43 @@ func Compile(compiler string, fileName string) (string, error) {
 		"Python": "#!/usr/bin/env python3",
 	}
 
-	logFile, _ := os.OpenFile("error.log", os.O_RDWR|os.O_CREATE|os.O_APPEND, 0600)
-	defer logFile.Close()
-	log.SetOutput(logFile)
 	commands, compilerSupported := compilationCommand[compiler]
 	if compilerSupported == false {
-		log.Println("Linguagem de programação escolhida não suportada")
+		logger.Info().
```

Comment:
I know it was like this before, but maybe change this to english?

Reply:
Changed to english